example/test_problem/ELBDM/LSS_Hybrid/download_heavy_halo_ic.sh

```diff
@@ -0,0 +1,7 @@
+filename=Music_InitCondition_z99_L2.8_N0256_HeavyHalo
+link=http://use.yt/upload/4587e2e6
+
+curl ${link} -o ${filename}
+ln -sf ${filename} UM_IC_wave_heavy
+python elbdm_wave_to_hybrid.py -input UM_IC_wave_heavy -output UM_IC_hybrid_heavy -resolution 256
```

Comment:
To use a consistent file name and also avoid the error when using Python 2, I think it should be 
`python3 elbdm_wave_to_hybrid_IC.py` 
here (as the command in the `download_light_halo_ic.sh`).

Reply:
@JeiwHuang 
Fixed. :) But the script should work with Python 2. Do you encounter problems?

Reply:
@KunkelAlexander  Thanks for fixing it!
I would encounter this syntax error if I used my default `python` on eureka, which is Python 2.7.18.
```
  File "elbdm_wave_to_hybrid_IC.py", line 32
    print(str(sys.argv[t]), end=' ')
                               ^
SyntaxError: invalid syntax

```

Reply:
@JeiwHuang 
True, the default Python 2 and Python 3 syntax for printing without a newline is different. Then let us just stick to Python 3.